helpdesk_mgmt_project/migrations/16.0.2.2.0/post-migration.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    """Set the default_project_id value in the project company."""
+    env.cr.execute(
+        """
+        SELECT team.id, team.old_default_project_id, project.company_id
+        FROM helpdesk_ticket_team AS team
+        JOIN project_project AS project ON team.old_default_project_id = project.id
```

Comment:
Use normalized abbreviations from table names for homogeneity.

```suggestion
        FROM helpdesk_ticket_team htt
        JOIN project_project pp ON htt.old_default_project_id = pp.id
```